oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -44,10 +45,21 @@ struct Node {
     // TODO(#630): Remove exception when label tracking is implemented.
     #[allow(dead_code)]
     label: oak_abi::label::Label,
+
+    /// A [`HashSet`] containing all the handles associated with this Node.
+    handles: Mutex<HashSet<Handle>>,
```

Comment:
(Aside: I still don't really understand how handles map to channels here; the equivalent data structure in the C++ runtime has a per-Node map from handle value to channel reference, not just a set.)